### PR TITLE
Feature/course 2

### PR DESCRIPTION
**Pull request by davide-st**

### CODE REVIEW

File: staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        <p>
+
+      </div>
+  </div>
+  </footer>
+
+  </div>
+  <script src="pro-congress-113-senate.js"></script>
+  <script src="main-senate.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
```
Comment (manuelbarzi) [inline staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html:95]: keep third-party libraries tag declarations above your own scripts. move this bootstrap bundle file 2 lines up.

File: staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          </tbody>
+        </table>
+      </div>
+    </section>
+
+    <footer class="footer">
+      <div class="container-fluid">
+
+
+        <p class="footer"> &#169 2016 TGIF | All Rights Reserved
```
Comment (manuelbarzi) [inline staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html]: html entities (like `&copy;` for instance, the one you are using here) should have a trailing semicolon (`;`)

File: staff/davide-stefani/course-2/task-2/plan-of-attack/9/house-data.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        <p>
+
+      </div>
+  </div>
+  </footer>
+
+  </div>
+  <script src="pro-congress-113-house.js"></script>
+  <script src="main-house.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
```
Comment (manuelbarzi) [inline staff/davide-stefani/course-2/task-2/plan-of-attack/9/house-data.html]: move this third-party script above yours (two lines up ,)

File: staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html (hunk 1 above)
Comment (davide-st) [reply staff/davide-stefani/course-2/task-2/plan-of-attack/9/senate-data.html:95]: ok

File: staff/davide-stefani/course-2/task-2/plan-of-attack/9/house-data.html (hunk 3 above)
Comment (davide-st) [reply staff/davide-stefani/course-2/task-2/plan-of-attack/9/house-data.html]: done

**Comment by manuelbarzi** [comment]
There has been a change in the `upstream` repo, please, re-do this PR against your new branch `develop-<your-program-year-month`>. Thank you!

**Pull request closed without merging**